xgboost/src/main/java/hivemall/xgboost/XGBoostUDTF.java

```diff
@@ -269,44 +270,35 @@ public void checkTargetValue(double target) throws HiveException {}
     public void process(Object[] args) throws HiveException {
```

Comment:
I guess so.. 

Or, you can import `resources/eclipse-style.xml` for each project setting in IntelliJ.